app/controllers/messages_controller.rb

```diff
@@ -13,30 +18,33 @@ def index
 	#Displays all messages for a specific country
 	def show
 		@messages = Message.where(recipient: params[:id]).order('created_at DESC')
-		
+
 		respond_to do |format|
 			format.html
 			format.json { render json: @messages }
 		end
 	end
 
-
 	def new
 		@message = Message.new
 	end
 
-
 	def create
-		@message = Message.new(message_params)
 		@game = current_game
+
+		recipientId = message_params['recipient'].to_i
+		senderId = message_params['sender'].to_i
+
+    new_params = message_params.except('recipient', 'sender')
```

Comment:
This is not actually indented incorrectly. Github is having trouble with this file for some reason.